packages/preview/package.json

```diff
@@ -8,7 +8,7 @@
     "@theia/languages": "^0.12.0",
     "@theia/mini-browser": "^0.12.0",
     "@types/highlight.js": "^9.12.2",
-    "@types/markdown-it": "^0.0.4",
+    "@types/markdown-it": "*",
```

Comment:
Are we sure we want to match any version? `*`

Reply:
please use `^` where possible to allow only minor fixes and `~` or exact match to overcome bugs

Reply:
In this case, `^0.0.4` is semantically identical to `0.0.4` because of the major and minor `0`.

On the other hand, `@types/markdown-it-anchor` targets `*` too, leading to type mismatches when we upgrade this package.

I think it is fine for this particular case.

Reply:
so it just to avoid installing 2 different versions? makes sense then